Mathlib/RingTheory/PowerSeries/Order.lean

```diff
@@ -135,6 +136,15 @@ theorem order_eq {φ : R⟦X⟧} {n : ℕ∞} :
   | top => simp
   | coe n => simp [order_eq_nat]
 
+theorem order_coe {φ : R⟦X⟧} : φ.order = MvPowerSeries.order φ := by
```

Comment:
There are no coercion happening here.
The name should perhaps just be `order_eq_order`?